etc/chaos_supervisor.conf

```diff
@@ -1,5 +1,5 @@
 [program:chaos]
-command=/root/.virtualenvs/chaos/bin/python chaos.py
+command=bash chaos_wrapper.sh
```

Comment:
Surprisingly, it doesn't. That's actually what I had originally, and I had to fiddle with it to get it to work. I suspect that supervisord is actually `exec`-ing the command or something in a way that doesn't know about interpreters...